sandbox/OPEN-SOURCE.md

```diff
@@ -0,0 +1,29 @@
+# Open Source
+
+Defense Unicorns values Open Source. We understand that Open Source is a way for all members of the company to prevent silos and work in a collaboration-first mentality. 
+
+## Advocates
+- Brandt Keller
+
+## Why?
```

Comment:
It might be worth adding that not only are we sharing our knowledge (internally and externally) but we're also making it easier for us to pick up on knowledge and advancements from outside the company.

By taking part in the wider open source communities you end up hearing about new tools, technologies, and techniques a lot faster than you would have if you weren't actively paying attention.

Not sure if this would best in the `why` or `internal impact` section.

Reply:
Under the `#Why?` section, I propose we build an ordered list of FOSS principles that resonate with Defense Unicorns, each independently focusing on it's purpose / supporting value.

```
# Why?

Convergence & Collaboration: We believe that we are better together and these factors stand above all others to intensely accelerate mission outcomes. We relentlessly pursue alignment and partner where it exists to solve problems that no one can solve alone while enabling others to contribute in the future. We are on mission to destroy silos with the infectious culture of collaboration-first.

Transparency: We believe that thoughts, ideas and work should be accessible to build upon one another's contributions, formulate new ideas/improve existing ideas, make more effective decisions, understand how decisions affect us and iterate at speed.
```

Reply:
Good source of principles: https://agilemanifesto.org/principles.html